basics.md

```diff
@@ -22,7 +22,7 @@ a standalone executable.
 
 `import "fmt"` says we need `fmt` in addition to `main` <2>. A package other
 than `main` is commonly called a library, a familiar concept in many programming
-languages (see (#packages)). The line ends with a comment that begins with `//`.
+languages (see [Packages](#packages)). The line ends with a comment that begins with `//`.
```

Comment:
This should not be needed, `(#packages)` is sufficient. And the the html is work, for mmarks RFC output this should also be enough